packages/async-ssr-manager/src/__tests__/use-fake-timers.ts

```diff
@@ -1,46 +1,45 @@
 export type Effect<TResult> = () => Promise<TResult>;
 
-class ObservablePromise<TResult> {
+class MonitorablePromise<TResult> {
   public readonly promise: Promise<TResult>;
 
-  private _pending = true;
+  public pending = true;
 
   public constructor(promise: Promise<TResult>) {
     this.promise = (async () => {
       try {
         return await promise;
       } finally {
-        this._pending = false;
+        this.pending = false;
       }
     })();
   }
-
-  public async isPending(): Promise<boolean> {
-    for (let i = 0; i < 25; i += 1) {
-      await Promise.resolve();
-    }
-
-    return this._pending;
-  }
 }
 
+const queueMicroTask = process.nextTick.bind(process);
```

Comment:
While this shouldn't make any difference here, technically the next tick queue is before micro tasks so `(f) => Promise.resolve.then(f)` should be more correct, shouldn't it?